aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -502,28 +506,19 @@ impl AptosVMImpl {
             .or_else(|err| convert_prologue_error(transaction_validation, err, log_context))
     }
 
-    /// Run the epilogue of a transaction by calling into `EPILOGUE_NAME` function stored
-    /// in the `ACCOUNT_MODULE` on chain.
-    pub(crate) fn run_success_epilogue(
+    fn run_epiloque(
         &self,
         session: &mut SessionExt,
         gas_remaining: Gas,
         txn_data: &TransactionMetadata,
-        log_context: &AdapterLogSchema,
-    ) -> Result<(), VMStatus> {
-        fail_point!("move_adapter::run_success_epilogue", |_| {
-            Err(VMStatus::Error(
-                StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR,
-                None,
-            ))
-        });
-
-        let transaction_validation = self.transaction_validation();
+        transaction_validation: &TransactionValidation,
+    ) -> VMResult<()> {
         let txn_sequence_number = txn_data.sequence_number();
         let txn_gas_price = txn_data.gas_unit_price();
         let txn_max_gas_units = txn_data.max_gas_amount();
-        session
-            .execute_function_bypass_visibility(
+        if txn_sequence_number & GAS_PAYER_FLAG_BIT == 0 {
```

Comment:
Can you add comments here to explain that this doesn't need a feature flag guarding because prologue should already check so a tx would have failed there if they try to use the feature before it's enabled?